anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-publish.adoc

```diff
@@ -97,26 +97,30 @@ You might want to set a unique custom message ID instead of letting Anypoint MQ
 
 === Set a Custom Message ID
 
-Instead of letting the Anypoint MQ service generate a unique `messageId` value for the new published message, you can set a custom `messageId` value for your outgoing message using the *Message Id* field in the *Advanced* tab of the main screen:
+Instead of letting Anypoint MQ generate a unique `messageId` value for the new published message, set a custom `messageId` value in the *Message Id* field in the *Advanced* tab:
 
-image::amq-3x-publish-advanced.png[Anypoint MQ Publish Advanced Tab]
+image::amq-3x-publish-advanced.png[Anypoint MQ Publish Advanced Tab - Message Id]
 
-The ID of an Anypoint MQ message must be unique. Make sure to choose a unique custom ID to avoid unwanted side effects of duplicated IDs. In FIFO queues, messages with duplicate IDs are overwritten.
+The ID of an Anypoint MQ message must be unique. Make sure to choose a unique custom ID to avoid unwanted side effects of duplicated IDs.
+FIFO queues provide message deduplication. If multiple messages are sent to a FIFO queue containing the same message ID, the first message to arrive is kept as the valid message.
+See xref:mq::mq-queues.adoc#fifo-exactly-once-delivery[FIFO Exactly-Once Delivery] for more information.
```

Comment:
This is just a comment. I'm using *refer to* instead of *see* for accessibility purposes.

Reply:
Yeah, we discussed this word as potentially non-inclusive (based on a comment in a Write the Docs presentation) and ultimately decided "see" is fine. We still use "See Also" headings, and so does Salesforce CX.